webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -106,6 +116,51 @@ class ResponsePage extends Component {
         };
     };
 
+    renderReviewResponse(review_response, section) {
+        const questions = section.review_questions.map(q => {
+            const a = review_response.scores.find(a => a.review_question_id === q.id);
+            if (q.type === "information") {
+                return <h4>{q.headline}</h4>
+            };
```

Comment:
So "information" questions in the review form are used to refer back to answers in the original response, so that the reviewer can see what the candidate's answers were in the review form. Now on the response page, you of course already have the full response, so you don't need these information questions and can instead just filter them out. One simple way to do it is to just not display anything where you can't find `a`.